<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSCalibExtract</class>
 <widget class="QMainWindow" name="SSCalibExtract">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="3,0,0,0,0,0,0,0">
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="upperFrequencyCutOffSpinBoxWidget">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="value">
       <double>4500.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Upper frequency cutoff</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="executeText"/>
      </item>
      <item>
       <widget class="QPushButton" name="executePushButtonWidget">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="3">
     <layout class="QGridLayout" name="guiCanvas"/>
    </item>
    <item row="7" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="saveFileButtonWidget">
        <property name="text">
         <string>Save calibration</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Pump wavelength</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="undoButtonWidget">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="pumpWavelengthSpinBoxWidget">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="value">
       <double>800.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="saveFileDefaultButtonWidget">
      <property name="text">
       <string>Save calibration default</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="loadFileButtonWidget">
      <property name="text">
       <string>Load file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Lower frequency cutoff</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="lowerFrequencyCutOffSpinBoxWidget">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="value">
       <double>300.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
